docs/pages/admin-guides/infrastructure-as-code/terraform-provider/terraform-cloud.mdx

```diff
@@ -19,6 +19,10 @@ dedicated guide:
   Terraform Cloud joining with self-hosted Terraform Enterprise requires
   Teleport Enterprise. Terraform Cloud joining with public HCP Terraform
   (https://app.terraform.io) is supported in Teleport Community Edition.
+
+  Additionally, note that Terraform Enterprise joining is currently in Preview.
+  Please reach out [Teleport support](https://goteleport.com/support) if you
+  encounter issues.
```

Comment:
I'm not sure if there's a real "process" around marking things as Preview, but given the state of the feature I wanted to make it clear it may have issues.

Reply:
I would just remove this. People should reach out to support any time they encounter issues, and we've largely gotten away from the preview program.

Reply:
Fair enough, removed!